addons/humor/yeetgun/config.cpp

```diff
@@ -0,0 +1,117 @@
+class CfgPatches 
+{
+    class cnto_humor 
+    {
+        units[] = {};
+        weapons[] = {"cnto_hgun_Pistol_Yeetgun"};
+        requiredAddons[] = {
+            "A3_Weapons_F_Pistols_Pistol_Heavy_02",
+            "A3_Weapons_F",
+            "A3_Sounds_F",
+            "A3_Sounds_F_Enoch"
+        };
+    };
+};
+
+class CfgFunctions
+{
+    class cnto_yeetgun
+    {
+        class all
+        {
+            file = "cnto\additions\humor\yeetgun";
+            class yeetGunFired;
+        }
+    };
+};
+
+class Mode_SemiAuto;
+class CfgWeapons
+{
+    class hgun_Pistol_heavy_02_F;
+    class cnto_hgun_Pistol_Yeetgun: hgun_Pistol_heavy_02_F 
+    {
+        author = "Seb";
+        baseWeapon = "cnto_hgun_Pistol_Yeetgun";
+        picture = "\A3\Weapons_F_EPA\Pistols\Pistol_Heavy_02\data\UI\gear_Pistol_heavy_02_X_CA.paa";
+        magazines[] = {"Cnto_Yeetgun_Cannister"};
+        magazineWell[] = {""};
+        displayname = "The Yeet Gun™";
+        descriptionShort = "Handgun..?<br />Caliber: Air";
+        hiddenSelectionsTextures[] = {"\cnto\additions\humor\yeetgun\cnto_hgun_Pistol_Yeetgun","\A3\Weapons_F_EPA\Pistols\Pistol_Heavy_02\data\Pistol_Heavy_02_mag_co"};
+        class Library
+        {
+            libTextDesc = "Nobody is really sure how this works. The inventor was killed when this weapon was fired for the first time as his house collapsed in on him. RIP big G.";
+        };
+        class EventHandlers
+        {
+            fired = "_this call cnto_yeetgun_fnc_yeetGunFired";
+        };
+        class Single: Mode_SemiAuto
+        {
+            class BaseSoundModeType;
+            class StandardSound: BaseSoundModeType
+            {
+                soundSetShot[] = {"Yeetgun_Shot_SoundSet","Zubr_Tail_SoundSet","Zubr_InteriorTail_SoundSet"};
+            };
+            class SilencedSound: BaseSoundModeType
+            {
+            };
+        };
+    };
+};
+
+class CfgMagazines
+{
+    class 6Rnd_45ACP_Cylinder;
+    class Cnto_Yeetgun_Cannister: 6Rnd_45ACP_Cylinder 
```

Comment:
That uppercase `C` inconsistency (in `Cnto`) just rubs me the wrong way when we use `cnto_` everywhere else.